examples/webgl_loader_fbx.html

```diff
@@ -0,0 +1,203 @@
+				stats = new Stats();
+				stats.domElement.style.position = 'absolute';
+				stats.domElement.style.top = '0px';
+                container.appendChild( stats.domElement );
+
+
+                //////////////////////////////////////////////////////////////
+				// model
+				var manager = new THREE.LoadingManager();
+				manager.onProgress = function ( item, loaded, total ) {
+
+					console.log( item, loaded, total );
+
+				};
+
+				var onProgress = function ( xhr ) {
+					if ( xhr.lengthComputable ) {
+						var percentComplete = xhr.loaded / xhr.total * 100;
+						console.log( Math.round(percentComplete, 2) + '% downloaded' );
+					}
+				};
+
+				var onError = function ( xhr ) {
+				};
+
+                var loader = new THREE.FBXLoader( manager );
+				loader.load( 'models/fbx/xsi_man_skinning.fbx', function ( object ) {
+
+					object.traverse( function ( child ) {
+
+						if ( child instanceof THREE.Mesh ) {
+
+							// pass
+
+                        }
+
+						if ( child instanceof THREE.SkinnedMesh ) {
+
+                            if ( child.geometry.animations !== undefined || child.geometry.morphAnimations !== undefined ) {
+
+                                child.mixer = new THREE.AnimationMixer( child );
+                                mixers.push( child.mixer );
+
+                                var action = child.mixer.clipAction( child.geometry.animations[0] );
+                                action.play();
+
+                            }
+
+
+                        }
+
+
+					} );
+                    scene.add( object );
+
+
+                }, onProgress, onError );
+
+                //////////////////////////////////////////////////////////////
+				renderer = new THREE.WebGLRenderer();
+				renderer.setPixelRatio( window.devicePixelRatio );
+                renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.setClearColor( 0x000000 );
+                container.appendChild( renderer.domElement );
+
+                //////////////////////////////////////////////////////////////
+                // controls, camera
+                controls = new THREE.OrbitControls( camera, renderer.domElement );
+                controls.target.set( 0, 12, 0 );
+				camera.position.set( 2, 18, 28 );
+				camera.lookAt( new THREE.Vector3( 0, 15, 0 ) );
+
+				window.addEventListener( 'resize', onWindowResize, false );
+
+                animate();
+			}
+
+			function onWindowResize() {
+
+				camera.aspect = window.innerWidth / window.innerHeight;
+				camera.updateProjectionMatrix();
+
+				renderer.setSize( window.innerWidth, window.innerHeight );
+
+			}
+
+			//
+            function animate() {
+
+                requestAnimationFrame( animate );
+
+                if( mixers.length > 0 ){
+                    for ( var i=0; i < mixers.length; i++ ){
+                        mixers[i].update( clock.getDelta() );
+
+                    }
+
+                }
+
+				controls.update();
```

Comment:
`contols.update()` should not be required for `OrbitControls` unless `enableDamping` is true.